pkg/controller/projectreference/projectreference_adapter.go

```diff
@@ -85,61 +85,61 @@ var supportedRegions = map[string]bool{
 
 //ReferenceAdapter is used to do all the processing of the ProjectReference type inside the reconcile loop
 type ReferenceAdapter struct {
-	projectClaim     *gcpv1alpha1.ProjectClaim
-	projectReference *gcpv1alpha1.ProjectReference
+	ProjectClaim     *gcpv1alpha1.ProjectClaim
+	ProjectReference *gcpv1alpha1.ProjectReference
 	logger           logr.Logger
 	kubeClient       client.Client
 	gcpClient        gcpclient.Client
 }
 
-func newReferenceAdapter(projectReference *gcpv1alpha1.ProjectReference, logger logr.Logger, client client.Client, gcpClient gcpclient.Client) (*ReferenceAdapter, error) {
+func NewReferenceAdapter(projectReference *gcpv1alpha1.ProjectReference, logger logr.Logger, client client.Client, gcpClient gcpclient.Client) (*ReferenceAdapter, error) {
```

Comment:
Needs a comment to above the function `// NewReferenceAdapater ....`

Reply:
I dislike this so much.